includes/view/ShiftEntry_view.php

```diff
@@ -224,8 +224,8 @@ function ShiftEntry_edit_view(
                 config('max_freeloadable_shifts')]);
         }
         $freeload_form = [
-            form_checkbox('freeloaded', __('Freeloaded') . ' <span class="bi bi-info-circle-fill text-info" data-bs-toggle="tooltip" title="' .
-                $freeload_info . '"></span>', $freeloaded),
+            form_checkbox('freeloaded', (!$freeloaded_by ? __('Freeloaded') : __('Freeloaded by %s', [User_Nick_render(User::find($freeloaded_by->id))])) . ' <span class="bi bi-info-circle-fill text-info" data-bs-toggle="tooltip" title="' .
```

Comment:
`$freeloaded_by` is an ID that should be used directly in find
Also splitting it to multiple lines again would be great ;)